go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -49,20 +54,37 @@ type EmergencyReparenter struct {
 	logger logutil.Logger
 }
 
-// EmergencyReparentOptions provides optional parameters to
-// EmergencyReparentShard operations. Options are passed by value, so it is safe
-// for callers to mutate and reuse options structs for multiple calls.
-type EmergencyReparentOptions struct {
-	NewPrimaryAlias     *topodatapb.TabletAlias
-	IgnoreReplicas      sets.String
-	WaitReplicasTimeout time.Duration
-
-	// Private options managed internally. We use value passing to avoid leaking
-	// these details back out.
+type (
+	// EmergencyReparentOptions provides optional parameters to
+	// EmergencyReparentShard operations. Options are passed by value, so it is safe
+	// for callers to mutate and reuse options structs for multiple calls.
+	EmergencyReparentOptions struct {
+		newPrimaryAlias           *topodatapb.TabletAlias
+		ignoreReplicas            sets.String
+		waitReplicasTimeout       time.Duration
+		preventCrossCellPromotion bool
+
+		lockAction string
+	}
+)
 
-	lockAction string
+// NewEmergencyReparentOptions creates a new EmergencyReparentOptions which is used in ERS
```

Comment:
I'm not sure I see the value in this function vs making callers inline the struct themselves

Reply:
Fixed